.github/workflows/release-cross-repo-test.yml

```diff
@@ -0,0 +1,61 @@
+name: Release Ansible cross repository test
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [opened]  # Workflow will only be executed when PR is opened to main branch
+  workflow_dispatch: # Manual trigger
+
+
+jobs:
+  ansible_integration_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout linode_api4 repository
+        uses: actions/checkout@v4
+
+      - name: update packages
+        run: sudo apt-get update -y
+
+      - name: install make
+        run: sudo apt-get install -y build-essential
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Install linode_api4
+        run: make install
+
+      - name: checkout repo
+        uses: actions/checkout@v3
+        with:
+          repository: linode/ansible_linode
+          path: .ansible/collections/ansible_collections/linode/cloud
+
+      - name: install dependencies
+        run: |
+          cd .ansible/collections/ansible_collections/linode/cloud
+          pip install -r requirements.txt -r requirements-dev.txt --upgrade-strategy only-if-needed
```

Comment:
Just for some added confidence, would it make sense to install `linode_api4` after installing the Ansible Collection deps?

Reply:
I verified that I am using the one that is built on my local.
```
╰─➤  pip install linode_api4 --upgrade-strategy only-if-needed
Requirement already satisfied: linode_api4 in ./venv/lib/python3.10/site-packages (0.0.0.dev0)
Requirement already satisfied: requests in ./venv/lib/python3.10/site-packages (from linode_api4) (2.31.0)
```
 I think doing it after might be complicated considering it will be overwritten